packages/ERTP/src/types.js

```diff
@@ -183,7 +183,7 @@
 /**
  * @callback MakeIssuerKit
  * @param {string} allegedName
- * @param {AmountMathKind} amountMathKind
+ * @param {AmountMathKind?} amountMathKind
```

Comment:
```suggestion
 * @param {AmountMathKind=} amountMathKind
```
? will allow null but I think we actually want undefined

Reply:
@katelynsills, It feels like I've gotten the opposite advice previously. Is there a doc I can refer to?

Reply:
Hmm, I was able to mouse-over the definition and see that it allowed for null. If you change it to `=` it will show that it allows for undefined instead.